src/ja/server/database/sql/database.py

```diff
@@ -177,6 +182,10 @@ def find_job_by_id(self, job_id: str) -> Optional[DatabaseJobEntry]:
                 jobs_entry.statistics.paused_time \
                     = (datetime.now() - jobs_entry.statistics.time_started).seconds - jobs_entry.statistics.running_time
             session.commit()
+            logger.info("job entry with job id: %s found." % job_id)
+            logger.debug("%s" % str(jobs_entry.job.to_dict()))
```

Comment:
Couldn't you just use str(jobs_entry.job) since Job has implemented __str__() ?

Reply:
yea probably, sorry for the late replay I broke my linux groups on my laptop, and after some trying I reinstalled, at least all the tests are passing locally now.